lib/plugins/create/templates/aws-nodejs/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "aws-nodejs",
+  "version": "1.0.0",
+  "description": "",
+  "main": "handler.js",
+  "scripts": {
+    "test": "jest"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "babel-jest": "^19.0.0",
+    "babel-preset-es2015": "^6.24.0",
+    "babel-preset-stage-0": "^6.22.0",
```

Comment:
https://github.com/babel/babel-preset-env is now a recommended official approach